cgi-bin/Apache/BML.pm

```diff
@@ -1916,7 +1916,7 @@ sub get_path_info
 sub get_remote_ip
 {
     my $apache_r = BML::get_request();
```

Comment:
Might want to see how many uses there are, and how many of them are in BML files.